cotainr/tests/container/patches.py

```diff
@@ -56,6 +56,17 @@ def mock_enter(self):
 
         return ret_val
 
+    def outside_container_create(self, env_file):
```

Comment:
The [naming convention here](https://cotainr.readthedocs.io/en/stable/development/test_suite_ci_cd.html#the-tests-folder-hierarchy-and-naming-conventions) is to prepend the function name with `patch_`.

Reply:
True